### ISSUE TITLE
[zf-apigility-doctrine] Doctrine configuration broken

### ISSUE THREAD

**Issue by MauricioKruijer** [MauricioKruijer/Apigility-bookshelf#7]
After installing https://github.com/zfcampus/zf-apigility-doctrine/releases/tag/2.3.0 I ran into an error while creating a new REST resource for my API.

![Setting up new REST Service modal](https://user-images.githubusercontent.com/224625/61299750-fb2c2d00-a7e0-11e9-8b83-9bbd5dc3853e.png)

> Doctrine configuration broken 
> You have not yet configured any Doctrine connection, and thus cannot create a Doctrine-Connected service.
> Please refer to the documentation page and define validation configuration for your Doctrine connection.

The documentation link shown in the modal seems to be broken.

**Comment by MauricioKruijer** [comment]
To fix this issue I had to install another dependency `composer require doctrine/doctrine-orm-module` 
Resources I find really helpful in figuring this out (since I'm not familiar with ZF2, Doctrine or Apigility at all)
- https://github.com/zfcampus/zf-apigility-doctrine/issues/192 Thanks @santiph and @jguittard
- https://github.com/zfcampus/zf-apigility-doctrine/issues/268 Thanks @webimpress
- https://github.com/webimpress/zf3-apigility-doctrine/blob/master/config/autoload/user.global.php
- https://github.com/webimpress/zf3-apigility-doctrine/blob/master/module/Application/src/Entity/Album.php

I am now a step closer to my goal! haha

**Comment by MauricioKruijer** [comment]
Alright, maybe I celebrated the achievement a bit toooooo soon XD

I thought, it would be way nicer to have my entities in the right Namespace / Module. So I went full yolo and added this to doctrine.local.php
```PHP
<?php

use Doctrine\ORM\Mapping\Driver\AnnotationDriver;

return [
    'doctrine' => [
        'connection' => [
            'orm_default' => [
                'driverClass' => 'Doctrine\DBAL\Driver\PDOMySql\Driver',
                'params'      => [
                    'charset'  => 'utf8',
                    'host'     => 'localhost',
                    'port'     => '3306',
                    'user'     => 'bookshelf',
                    'password' => 'bookshelf',
                    'dbname'   => 'bookshelf',
                ],
            ],
            'bookshelf' => [
                'driverClass' => 'Doctrine\DBAL\Driver\PDOMySql\Driver',
                'params'      => [
                    'charset'  => 'utf8',
                    'host'     => 'localhost',
                    'port'     => '3306',
                    'user'     => 'bookshelf',
                    'password' => 'bookshelf',
                    'dbname'   => 'bookshelf',
                ],
            ],
        ],
        'driver'     => [
            'Doctrine_driver' => [
                'class' => AnnotationDriver::class,
                'cache' => 'array',
                'paths' => [
                    __DIR__ . '/../../module/Application/src/Entity',
                ],
            ],
            'Bookshelf_driver' => [
                'class' => AnnotationDriver::class,
                'cache' => 'array',
                'paths' => [
                    __DIR__ . '/../../module/Bookshelf/src/Entity',
                ],
            ],
            'orm_default' => [
                'drivers' => [
                    'Application\\Entity' => 'Doctrine_driver',
                ],
            ],
            'bookshelf' => [
                'drivers' => [
                    'Bookshelf\\Entity' => 'Bookshelf_driver',
                ],
            ],
        ],
    ],
];
```

Looks good doesn't it?

Yeah nope...

## After selecting this new Entity Manager
<img width="744" alt="Screenshot 2019-07-16 at 21 24 36" src="https://user-images.githubusercontent.com/224625/61323595-a99b9680-a810-11e9-8d0c-2f0f03da52ad.png">

## This annoying error pops-up (again!)
<img width="440" alt="Screenshot 2019-07-16 at 21 23 42" src="https://user-images.githubusercontent.com/224625/61323633-c6d06500-a810-11e9-844c-b7c3f001e3fc.png">


# Todo
Add Author entity in Application realm!

**Comment by MauricioKruijer** [comment]
# Moved Bookshelf/Entity to Application/Entity (namespace and directory)
<img width="746" alt="Screenshot 2019-07-16 at 21 44 04" src="https://user-images.githubusercontent.com/224625/61324487-da7ccb00-a812-11e9-99bd-01913c5c283e.png">

🎉 Its finally showing up as entity

> Add Author entity in Application realm!
Worked by the way

**Comment by MauricioKruijer** [comment]
Kind of resolved by #8 👍
